nessie/jobs/refresh_canvas_data_catalog.py

```diff
@@ -97,13 +97,14 @@ def run(self):
 
         if redshift.execute_ddl_script(canvas_external_catalog_ddl):
             app.logger.info(f'Canvas schema creation job completed.')
-            return self.verify_external_data_catalog()
+            if self.verify_external_data_catalog():
+                return 'Canvas schema verfication job completed.'
```

Comment:
@sandeepmjay Do you need some code for the `else` condition here? (If `redshift.execute_ddl_script(canvas_external_catalog_ddl)` returns true but `self.verify_external_data_catalog()` returns false).

Reply:
The verify external data catalog raises a background job error in case of verification failures.

Reply:
Thanks, looks good.